partiql-ast/build.gradle.kts

```diff
@@ -16,6 +16,9 @@
 plugins {
     id(Plugins.conventions)
     id(Plugins.publish)
+    // Need the Kotlin lombok plugin to allow for Kotlin code in partiql-ast to understand Java Lombok annotations.
+    // https://kotlinlang.org/docs/lombok.html
+    id(Plugins.kotlinLombok) version Versions.kotlinLombok
```

Comment:
(self-review): Kotlin lombok plugin required to allow Kotlin code to understand Java lombok annotations -- https://kotlinlang.org/docs/lombok.html. It is experimental but 
- seems pretty stable so far since its published by the Kotlin team
- we could always use a fixed version or hand-write the getters if we encounter issues without breaking the public API